apps/meteor/client/views/root/hooks/useForceLogout.ts

```diff
@@ -0,0 +1,22 @@
+import { useUserId, useStream } from '@rocket.chat/ui-contexts';
+import { Session } from 'meteor/session';
```

Comment:
We have a hook called `useSessionDispatch` for setting meteor session.

Reply:
I was looking for something like this. Thanks man